dashboard/modules/log/log_agent.py

```diff
@@ -20,3 +26,130 @@ async def run(self, server):
     @staticmethod
     def is_minimal_module():
         return False
+
+
+# 64 KB
+BLOCK_SIZE = 1 << 16
+
+
+class LogAgentV1Grpc(
+    dashboard_utils.DashboardAgentModule, reporter_pb2_grpc.ReporterServiceServicer
+):
+    def __init__(self, dashboard_agent):
+        super().__init__(dashboard_agent)
+
+    async def run(self, server):
+        if server:
+            reporter_pb2_grpc.add_LogServiceServicer_to_server(self, server)
+
+    # TODO: should this return True
+    @staticmethod
+    def is_minimal_module():
+        return False
+
+    async def ListLogs(self, request, context):
+        logger.info(f"initiated ListLogs:\n{request}")
+
+        def on_exit(self):
```

Comment:
what's this for?

Reply:
I guess it is just debugging

Reply:
From discussion offline:
I think this kills two birds with one stone, making the gRPC handlers observable in the logs.
Testing the behaviour is only a small part of the usefulness.